src/libs/actions/App.ts

```diff
@@ -17,48 +16,56 @@ import * as SessionUtils from '@libs/SessionUtils';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import * as OnyxTypes from '@src/types/onyx';
+import type {OnyxData} from '@src/types/onyx/Request';
 import * as Policy from './Policy';
 import * as Session from './Session';
 import Timing from './Timing';
 
-let currentUserAccountID;
-let currentUserEmail;
+type PolicyParamsForOpenOrReconnect = {
+    policyIDList: string[];
+};
+
+type Locale = ValueOf<typeof CONST.LOCALES>;
+
+let currentUserAccountID: number | string;
```

Comment:
@neil-marcellini What do you think about making it to be just a `number` and default to `0`. With `null` it will require some updates [here](https://github.com/VickyStash/Expensify/blob/ts-migration/app-lib/src/libs/actions/App.ts#L463), like:
```
[currentUserAccountID ?? '']: {
```

Reply:
I don't think 0 is a good default because on my dev environment, and maybe for others, 0 is a special account with email `__FAKE__`. So I think null is a better default, or perhaps -1.

Reply:
You can fix this in a follow up if you like, I'll just merge what we have for now.